gatsby-node.js

```diff
@@ -0,0 +1,37 @@
+require('ts-node').register() // To use imports from ts files. See https://github.com/gatsbyjs/gatsby/issues/1457#issuecomment-446006181
+const path = require(`path`)
+const { getProjects } = require('./src/data/project')
+const R = require('ramda')
+
+exports.onCreateWebpackConfig = ({ actions }) => {
+  actions.setWebpackConfig({
+    node: { fs: 'empty' },
+  })
+}
+
+exports.createPages = async ({ actions }) => {
+  const { createPage } = actions
+
+  const germanTranslations = getProjects('de')
+  const englishTranslations = getProjects('en')
+
+  R.forEach(props => {
+    createPage({
+      path: `/de/project/${props.slug}`,
+      component: path.resolve(`src/templates/ProjectPageDE.jsx`),
```

Comment:
Would it have worked to use the component directly?

oh you need a locale context...